src/main/java/fact/datacorrection/InterpolateTimeSeries.java

```diff
@@ -36,6 +36,12 @@
     @Parameter(required = false, description = "The minimum number of neighboring pixels required for interpolation", defaultValue = "3")
     public int minPixelToInterpolate = 3;
 
+    @Parameter(required = false, description = "The key for the resulting badPixelSet.")
+    public String badPixelKey = "badPixels";
+
+    @Parameter(required = false, description = "The key to the unixTimeUTC of the Event.")
+    public String unixTimeKey = "UnixTimeUTC";
```

Comment:
Why is this a parameter?

Reply:
I use this processor twice for different inputs with different times one will use 'UnixTimeUTC' the other 'LONS_UnixTimeUTC'. This is due to the fact that i have to calibrate the pedestal that I superimpose.